src/e2e/utils/helpers.ts

```diff
@@ -160,3 +162,63 @@ export function removeUnicodeChars(text: string): string {
   // eslint-disable-next-line no-control-regex
   return text.replace(/[^\x00-\x7F]/g, "");
 }
+
+export const clickOnATagCheckDomain = async (
+  aTag: Locator,
+  host: string | RegExp = /.*/,
+  path: string | RegExp = /.*/,
+  page: Page,
+) => {
+  if (typeof host === "string")
+    host = new RegExp(escapeRegExp(host.replace(/^(https?:\/\/)/, "")));
+  if (typeof path === "string") path = new RegExp(".*" + path + ".*");
+
+  const href = await aTag.getAttribute("href");
+  if (href === null) return false;
+
+  await page.goto(href);
+  const currentUrl = new URL(page.url());
+  const perceivedHost = currentUrl.hostname;
+  const perceivedPath = currentUrl.pathname;
+  expect(host.test(perceivedHost)).toBeTruthy();
+  expect(path.test(perceivedPath)).toBeTruthy();
+  await page.goBack();
+};
+
+export const escapeRegExp = (str: string): string => {
+  return str.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+};
+
+export const forceLoginAs = async (
+  email: string,
+  password: string,
+  page: Page,
+  landingPage: LandingPage,
+  authPage: AuthPage,
+) => {
+  test.slow(
+    true,
+    "this test runs through the welcome scan flow, increasing timeout to address it",
+  );
+
+  // speed up test by ignoring non necessary requests
+  await page.route(/(analytics)/, async (route) => {
+    await route.abort();
+  });
+  await page.context().clearCookies();
```

Comment:
Is Playwright persisting cookies/state between tests?

Reply:
It would redirect me to dashboard whenever i'd try to go to the landingPage. If i clear the cookies it would get rid of the auth token, so I can start fresh.